spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-lattice/src/main/java/org/springframework/cloud/dataflow/module/deployer/lattice/LrpModuleDeployer.java

```diff
@@ -72,16 +76,19 @@ public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		Map<String, String> rawArgs = new HashMap<>();
 		rawArgs.putAll(request.getDefinition().getParameters());
 		rawArgs.putAll(request.getDeploymentProperties());
-		Map<String, String> qualifiedArgs = ModuleArgumentQualifier.qualifyArgs(0, rawArgs);
+		int index = 0;
+		Map<String, String> qualifiedArgs = ModuleArgumentQualifier.qualifyArgs(index, rawArgs);
```

Comment:
did you try checking in `rawArgs` instead? would be similar to the local deployer: https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-local/src/main/java/org/springframework/cloud/dataflow/module/deployer/local/LocalModuleDeployer.java#L73